hpx/parallel/datapar/execution_policy.hpp

```diff
@@ -24,18 +24,18 @@
 #include <type_traits>
 #include <utility>
 
-namespace hpx { namespace parallel { HPX_INLINE_NAMESPACE(v1)
+namespace hpx { namespace parallel { namespace execution
```

Comment:
Right, my misunderstanding - I mixed that up with the executor work I'm doing right now. Please disregard my first comment.

I assume, you are talking about moving (keeping) the `datapar` execution policies into a versioned namespace. I agree, that should probably be done.